src/site/content/en/learn/css/transitions/index.md

```diff
@@ -0,0 +1,252 @@
+
+You can use DevTools to experiment with different timing functions in real-time.
+
+{% Img src="image/eiKy1JcjHqPp3gaedjAQWjPJ8YK2/G06SHV4Dra1HqTAL9d6t.png", alt="Chrome DevTools visual transition timing editor", width="264", height="359" %}
+
+### transition-delay
+
+Use the [`transition-delay`](https://developer.mozilla.org/en-US/docs/Web/CSS/transition-delay) property to specify the time at which a transition will start. If `transition-duration` is not specified, transitions will start instantly because the default value is `0s`. This property accepts a time unit, for example seconds (`s`) or milliseconds (`ms`).
+
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'jOwyWep'
+} %}
+
+This property is useful for staggering transitions, achieved by setting a longer `transition-delay` for each subsequent element in a group.
+
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'yLXgeRQ',
+  height: 410
+} %}
+
+`transition-delay` is also useful for debugging. Setting the delay to a negative value can start a transition further into the timeline.
+
+### shorthand: transition
+
+Like most CSS properties, there is a shorthand version. [`transition`](https://developer.mozilla.org/en-US/docs/Web/CSS/transition) combines `transition-property`, `transition-duration`, `transition-timing-function`, and `transition-delay`.
+
+```css
+.longhand {
+  transition-property: transform;
+  transition-duration: 300ms;
+  transition-timing-function: ease-in-out;
+  transition-delay: 0s;
+}
+
+.shorthand {
+  transition: transform 300ms ease-in-out 0s;
+}
+```
+
+## What can and can't transition?
+
+When writing CSS, you can specify which properties should have animated transitions. See [this MDN list of animatable CSS properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_animated_properties).
+
+In general, it's only possible to transition elements that can have a "middle state" between their start and final states. For example, it's impossible to add transitions for `font-family`, because it's unclear what the "middle state" between `serif` and `monospace` should look like. On the other hand, it is possible to add transitions for `font-size` because its unit is a length that can be interpolated between.
+
+{% Img src="image/eiKy1JcjHqPp3gaedjAQWjPJ8YK2/1Lr1iC56nyUyYWWv6qF2.jpg", alt="Diagram of shapes transitioning smoothly from one state to another, and two lines of text in different fonts that cannot be transitioned smoothly", width="800", height="600" %}
+
+Here are some common properties you can transition.
+
+### Transform
+
+The [`transform`](https://developer.mozilla.org/en-US/docs/Web/CSS/transform) CSS property is commonly transitioned because it is a GPU-accelerated property that results in smoother animation that also consumes less battery. This property lets you arbitrarily scale, rotate, translate, or skew an element.
```

Comment:
```suggestion
The [`transform`](https://developer.mozilla.org/docs/Web/CSS/transform) CSS property is commonly transitioned because it is a GPU-accelerated property that results in smoother animation that also consumes less battery. This property lets you arbitrarily scale, rotate, translate, or skew an element.
```